### PR TITLE
Adding check for value or path key in add patch operation

### PR DESCRIPTION
**Pull request by JasonStiefel**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,9 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-
 ## [Unreleased]
 
+## [0.2.3] - 2020-09-22
```
Comment (rkaippully) [inline CHANGELOG.md]: You can leave this under "unreleased". The `release.sh` script takes care of bumping up the release version and updating it every where - project.clj, README, ChangeLog etc.

File: project.clj [hunk 2]
```diff
@@ -1,4 +1,4 @@
-(defproject org.clojars.rkaippully/scim-patch "0.2.2"
+(defproject org.clojars.rkaippully/scim-patch "0.2.3"
```
Comment (rkaippully) [inline project.clj]: As I mentioned above, no need to change this. This is all automated.

File: src/scim_patch/core.clj [hunk 3]
```diff
@@ -101,6 +101,9 @@
 
 (defn op-add
   [schema resource opr]
+  (when (not-any? #(contains? opr %) [:value :path])
```
Comment (rkaippully) [inline src/scim_patch/core.clj]: I think `op-replace` has the same bug. Can you fix that too, please?

**Review by rkaippully — CHANGES_REQUESTED** [review]

File: src/scim_patch/core.clj (hunk 3 above)
Comment (JasonStiefel) [reply src/scim_patch/core.clj]: Read [the RFC](https://tools.ietf.org/html/rfc7644#section-3.5.2.1) again and realized that add MUST have a "value" key, which causes other tests to fail. Will need to do additional fixes to the op-replace bug mentioned.

**Review by rkaippully — APPROVED** [review]

**Pull request merged**